plugins/rest/rest_handlers.go

```diff
@@ -302,14 +306,14 @@ func (plugin *Plugin) commandHandler(formatter *render.Render) http.HandlerFunc
 		reply := &vpe.CliInbandReply{}
 		err = ch.SendRequest(r).ReceiveReply(reply)
 		if err != nil {
-			err = fmt.Errorf("sending request failed: %v", err)
-			plugin.Log.Error(err)
-			formatter.JSON(w, http.StatusInternalServerError, err)
```

Comment:
Problem here was that `error` interface is not marsheable by json, but you can simple use `Error()` method to return string where needed: https://play.golang.org/p/9gc87fjTApl

Reply:
Well, yes that's what the PR is about. Not sure what you mean here.